presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -99,6 +99,16 @@ String
 
     Example type definitions: ``varchar``, ``varchar(20)``
 
+    SQL statements support string value usage as simple literals ``Hello
+    world``. To use Unicode literals within a string, you have to use the prefix
+    ``U&`` for the completed string and prefix the Unicode character itself with
```

Comment:
Typo: completed -> complete

However, I think it's better to say "prefix the string literal with ... "